src/OrchardCore.Modules/OrchardCore.Contents/Services/DefaultContentsAdminListFilter.cs

```diff
@@ -92,6 +92,11 @@ public async Task FilterAsync(ContentOptionsViewModel model, IQuery<ContentItem>
                 {
                     query.With<ContentItemIndex>(x => x.ContentType.IsIn(listableTypes.Select(t => t.Name).ToArray()));
                 }
+                else
+                {
+                    //We never list HtmlWidget in the Content Items List
+                    query.With<ContentItemIndex>(x => x.ContentType != "HtmlWidget");
```

Comment:
I think we should never list any ContentType items which have a Stereotype. I need to do a double check on that one.